s32160-pj-Pawel_Rohde/udostepnienia_dostepu_cwiczenia.txt

```diff
@@ -0,0 +1,236 @@
+├── Lab01
+│   ├── LICENSE
+│   └── README.md
+├── lista_plikow_z_literka_b_2.txt
+├── lista_plikow_z_literka_b.txt
+├── placki.txt
+├── wynik3_1.txt
+└── wynik3_2.txt
+
+
+*s32160@szuflandia22:~$ rm -r placki.txt
+s32160@szuflandia22:~$ tree
+.
+├── 01-zmienne-i-warunki-s32160
+│   ├── LICENSE
+│   └── README.md
+├── Lab01
+│   ├── LICENSE
+│   └── README.md
+├── lista_plikow_z_literka_b_2.txt
+├── lista_plikow_z_literka_b.txt
+├── wynik3_1.txt
+└── wynik3_2.txt
+
+
+Udalo sie usunac cudzy plik w moim katalogu, poniewaz mialem do tego uprawnienia.
+
+Program Midnight Commander ulatwia nawigowanie po plikach i mozna przeprowadzic w nim te same operacje, co w trybie tekstowym tylko w latwiejszy sposob.
+
+2.W utworzonej na swoim koncie strukturze podkatalogow przeprowadzic eksperymenty:
+
+-usuwajac wszelkie uprawnienia dostepu do katalogu biezacego
+
+-usuwajac wszelkie uprawnienia dostepu do katalogu nadrzednego (nadkatalogu).
+
+	-W jakich przypadkach mozemy wykonac wtedy polecenie cd ? 
+
+	-W jakich przypadkach mozemy wykonac polecenie chmod ? 
+
+	-Czy mozemy bezposrednio przeskoczyc do katalogu ABC/XYZ, jesli nie mamy uprawnienia wstepu do ABC, ale mamy do XYZ ?
+
+	-Czy mozemy takze wrocic korzystajac z polecnia cd - ?
+
+*s3216-@szuflandia22:~/lab01$ chmod 000 ./
+Usunalem wszelkie uprawnienia do katalogu biezacego.
+
+*s32160@szuflandia22:~/ulab01$ cd praca
+-bash: cd: praca: Brak dostępu
+Przy pomocy 'cd' nie moge przejsc do katalogu podrzednego.
+*s32160@szuflandia22:~/lab02$ cd ..
+s32160@szuflandia22:~
+Moge za to przejsc do katalogu nadrzednego.
+*s32160@szuflandia22:~/ cd lab02/
+-bash: cd: lab02/: Brak dostępu
+*s32160@szuflandia22:~ cd ~/ukos/lab02/
+-bash: cd: /home/PJWSTK/s32160/ukos/lab02/: Brak dostępu
+Nie moge przejsc do katalogu lab01 sciezka wzgledna ani sciezka bezwzgledna, a takze przy pomocy polecenia 'cd -', 
+poniewaz nie mam zadnych uprawnien do tego katalogu.
+
+*s32160@szuflandia22:~$ chmod 754 lab01/
+s30899@szuflandia22:~$
+Polecenie 'chmod' moge wykonac znajdujac sie w miejscu do ktorego mam uprawnienia np. katalogu domowym.
+
+*s32160@szuflandia22:~lab01$ chmod 000 ../
+Usunalem wszelkie uprawnienia do katalogu nadrzednego.
+
+*s32160@szuflandia22:~/lab12$ cd ..
+-bash: cd: ..: Brak dostępu
+Przy pomocy 'cd' nie moge przejsc do katalogu nadrzednego z powodu braku uprawnien.
+*s32160@szuflandia22:~/lab01$ cd praca/
+s32160@szuflandia22:~/lab01/praca$
+Moge za to przejsc do katalogu podrzednego, poniewaz mam tam domyslne ustawienia dostepu.
+*s32160@szuflandia22:~/lab01$ cd ~
+s2160@szuflandia22:~$
+Oczywiscie moge tez przejsc do katalogu domowego przy pomocy sciezki bezwzglednej.
+Nie moge wrocic do katalogu lab02 za pomoca sciezki bezglednej ani polecenie 'cd -', poniewaz nie mam dostepu do zawartosci
+katalogu lab01.
+
+Polecenie 'chmod' moge wykonac znajdujac sie w katalogu, do ktorego mam uprawnienia np. katalog domowy.
+
+*s32160@szuflandia22:~$ chmod u-x,g-x,a-x 01-zmienne-i-warunki-s32160
```

Comment:
Ten zapis jest dziwny. Odejmujesz osobno dla właściciela x, potem dla grupy odejmujesz także x.
Już to można było zapisać jako ug-x a na końcu wszędzie odejmujesz x niezależnie od tego w której sekcji się ten x znajduje czyli:
`a-x` === `ugo-x` === `u-x,g-x,o-x`